Content.Server/Info/ShowRulesCommand.cs

```diff
@@ -10,55 +10,46 @@
 namespace Content.Server.Info;
 
 [AdminCommand(AdminFlags.Admin)]
-public sealed class ShowRulesCommand : IConsoleCommand
+public sealed class ShowRulesCommand : LocalizedCommands
 {
-    [Dependency] private readonly INetManager _net = default!;
     [Dependency] private readonly IConfigurationManager _configuration = default!;
+    [Dependency] private readonly INetManager _net = default!;
     [Dependency] private readonly IPlayerManager _player = default!;
 
-    public string Command => "showrules";
-    public string Description => "Opens the rules popup for the specified player.";
-    public string Help => "showrules <username> [seconds]";
-    public async void Execute(IConsoleShell shell, string argStr, string[] args)
-    {
-        string target;
-        float seconds;
+    public override string Command => "showrules";
 
-        switch (args.Length)
+    public override async void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        if (args.Length is < 1 or > 2)
```

Comment:
this is legal C# but just know that i respect you less as a person for writing it